test/integration/visitor_can_add_item_to_cart_and_view_cart_test.rb

```diff
@@ -7,15 +7,26 @@ class VisitorCanAddItemToCartAndViewCartTest < ActionDispatch::IntegrationTest
                 price: 1000)
 
     visit items_path
-    assert page.has_content?('Cart: 0')
+
+    within(".cart-count") do
+      assert page.has_content?('0')
+    end
+
     click_link 'Add To Cart'
 
     assert page.has_content?('You now have 1 gnar possum in your cart.')
-    assert page.has_content?('Cart: 1')
+
+    within(".cart-count") do
+      assert page.has_content?('1')
+    end
 
     click_link 'Add To Cart'
     assert page.has_content?('You now have 2 gnar possums in your cart.')
-    assert page.has_content?('Cart: 2')
+
+    within(".cart-count") do
+      assert page.has_content?('2')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.